lib/diagnostic.js

```diff
@@ -7,31 +7,43 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-const min = function min () {
+const min = function min (num1, num2) {
+  for let( i = 0; i > 0; i--) {result[i] = min[i])
+
+}
 
 }
 
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
-const sum = function sum () {
+const sum = function(num1, num2) {
+  return num1 + num2
```

Comment:
Yup, but this function should be able to take an unlimited number of arguments. Using the `arguments` keyword inside your function would be a helpful way to get an array of all the arguments that were passed in.